### DISCUSSION TITLE
Why SLO is showing Not Configured

### DISCUSSION THREAD

**Discussion by sachin-cloudhero** [coroot/coroot#270] [Q&A] [answered]
Hello,

I've just started using coroot for my projects. Everything is configured properly and are showing values except for the SLO. It shows Not Configured.
![image](https://github.com/coroot/coroot/assets/143994984/d0d4f60e-23a7-4574-b0ed-9fd609362f39)
Where do i configure it?

**Comment by def** [comment, marked as answer]
By default, Coroot uses eBPF-based metrics as Service Level Indicators (SLIs). However, it appears that there are no relevant metrics for this particular app in your case. You can configure Service Level Objectives (SLOs) using custom Prometheus metrics. Simply click on "95%" here and configure the metric selectors.
